README.md

```diff
@@ -147,7 +155,7 @@ After `eggs update`:
 ```ts
 import * as colors from "https://deno.land/std@0.58.0/fmt/colors.ts";
 import * as bcrypt from "https://deno.land/x/bcrypt@v0.2.1/mod.ts"
-import * as eggs from "https://x.nest.land/eggs@0.1.3/mod.ts"
```

Comment:
why import eggs?

Reply:
Just a proof of concept that the update command will update that dependency.